dist/svg.select.js

```diff
@@ -307,7 +371,8 @@ SVG.extend(SVG.Element, {
 });
 
 SVG.Element.prototype.selectize.defaults = {
-    points: true,                            // If true, points at the edges are drawn. Needed for resize!
+    points: ['lt', 'rt', 'rb', 'lb', 't', 'r', 'b', 'l'],    // which points to draw, default all
+    pointsExclude: '',                       // easier option if to exclude few than rewrite all
```

Comment:
should be an empty array by default. Shouldnt it?

Reply:
I thought that default would be to select all !

But be aware that this `default` option is also used to filter users `option` too ;)
I would leave it with all included by default !

Reply:
Oh, I suppose you're talking about `pointsExlude` option, yes, that can be an empty array too

I've built it to be used in any format: comma separated list or an array, for both options: `points` or `pointsExclude`

Reply:
I will feel better with an empty array here because it makes the intention more clear. Also: You used an array for points :).

Reply:
Will change it together with the Readme, to be a complete update ;)